### PR TITLE
update document for building android sdk

### PR DESCRIPTION
**Pull request by dwSun**
update document for building android sdk.

### CODE REVIEW

**Comment by hanrui1sensetime** [comment]
Thanks for your contribution.
And before LGTM, you may need to fix lint, includes: `trim trailing whitespace` and `mdformat`.
You could click `ci/circleci: link` for details.
Any questions can ask for us.

**Comment by dwSun** [comment]
lint errors fixed.
No idea why readthedocs failed.

**Comment by lvhan028** [comment]
> lint errors fixed. No idea why readthedocs failed.

It is caused by markdown_table version. #813 will fix it.

**Review by hanrui1sensetime — APPROVED** [review]
LGTM.

File: docs/en/01-how-to-build/android.md [hunk 1]
```diff
@@ -78,18 +78,23 @@ export OPENCV_ANDROID_SDK_DIR=${PWD}/OpenCV-android-sdk
...
 git submodule update --init
 export NCNN_DIR=${PWD}
-mkdir -p build
-cd build
-cmake -DCMAKE_TOOLCHAIN_FILE=${NDK_PATH}/build/cmake/android.toolchain.cmake -DANDROID_ABI="arm64-v8a" -DANDROID_PLATFORM=android-30 -DNCNN_VULKAN=ON -DNCNN_DISABLE_EXCEPTION=OFF -DNCNN_DISABLE_RTTI=OFF ..
-make install
+
+export ANDROID_ABI=arm64-v8a
+
+mkdir -p build\_${ANDROID_ABI}
```
Comment (RunningLeon) [inline docs/en/01-how-to-build/android.md:90]: do we need to add this symbol `\` ?

Comment (dwSun) [reply docs/en/01-how-to-build/android.md:90]: It is added by mdformat automatically.

**Review by RunningLeon — APPROVED** [review]
LGTM

File: docs/en/01-how-to-build/android.md (hunk 1 above)
Comment (hanrui1sensetime) [reply docs/en/01-how-to-build/android.md:90]: > It is added by mdformat automatically.

You can use
```bash
mkdir -p build-${ANDROID_ABI}
```

**Review by hanrui1sensetime — APPROVED** [review]
LTGM.

**Comment by OpenMMLab-Q** [comment]
Dear dwsun,

First of all, we want to express our gratitude for your significant PR in the MMDeploy project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

If you are Chinese or have **WeChat**，welcome to join our community on WeChat. You can add our assistant ：**openmmlabwx**. Please add "**mmsig + Github ID**" as a remark when adding friends：）

We would also like to invite you to join our Special Interest Group (SIG) private channel on **Discord**, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. We look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

Thank you again for your contribution❤

Best regards！@dwsun

**Pull request merged**